### PR TITLE
Update KSTO logo

### PR DESCRIPTION
**Pull request by drewvolz**
Personally prefer the older image. This PR serves as a discussion.

Before | After
--|--
old | image

### CODE REVIEW

**Comment by rye** [comment]
Is this a change that was made upstream (i.e. by KSTO)?

**Comment by hawkrives** [comment]
Remove the outline

**Comment by drewvolz** [comment]
@rye it’s an upstream change, but we don’t have to adopt it.

**Comment by hawkrives** [comment]
@drewvolz where did you even find that logo? 

Edit: I ask because I can't see a logo anywhere on kstoradio.org…

Edit: Ah. https://www.kstoradio.org/wp-content/uploads/sites/1178/2018/02/dumpsterfire.png … which is loaded somewhere on the front page, but I can't find it still lol

**Comment by drewvolz** [comment]
Looks like you found it... that's the one.

**Comment by hawkrives** [comment]
Could you please:

- Remove the blue border
- Resize the image file to remove the transparent edges

Currently, there's no easy way to change the button color for the radio view; I'll take that on in a separate PR.

**Comment by drewvolz** [comment]
Updated the image to reflect that there is no border now.

File: source/views/streaming/radio/controller.js [hunk 1]
```diff
@@ -177,7 +178,8 @@ export class RadioControllerView extends React.PureComponent<Props, State> {
 					const logoSize = {width: logoWidth, height: logoWidth}
 
 					const root = [styles.root, sideways && landscape.root]
-					const logo = [styles.logo, logoSize]
+					const borderStyle = imageBorder ? styles.logoBorder : null
```
Comment (hawkrives) [inline source/views/streaming/radio/controller.js]: Inline this variable into the `logo` array, please. You can also shorten it by `imageBorder && styles.logoBorder`.

**Review by hawkrives — CHANGES_REQUESTED** [review]

**Review by hawkrives — APPROVED** [review]

**Review by hawkrives — DISMISSED** [review]
Please remove the commit with the original image and force-push.

**Review by rye — DISMISSED** [review]

**Review by hawkrives — DISMISSED** [review]
Could you split adding the image and the code changes into two commits, please?

**Review by hawkrives — APPROVED** [review]

**Pull request merged**